### PR TITLE
ADD: OPTIMIZES IMAGES & MODIFY IMAGE SOURCE PATHS 🚀

### LINKED ISSUE #241: 💡 [IMPROVE] - PAGE PERFORMANCE BY OPTIMIZING IMAGE ASSETS
| Section                    | Description |
|----------------------------|-------------|
| **Concise Description**    | The site uses `jpg` types of images, but the `jpg` images are always tend to be more larger in size so that the page takes more bandwith to load it |
| **Solution Proposed**      | My plan is to convert all those `jpg` images to `webp` to minimize the page load time and bandwith|
| **Additional Context**     | Unoptimized images severly affects page load time, because it may uncomfortable for the people who having a moderate internet connection|

### Checklists

- [x] I follow the Contributing Guidelines of this project.
- [x] I have checked the current issues for duplicate problems.
- [x] I have read the readme of this repository.
- [x] I am from `GSSoC'23`.
- [x] I would like to be assigned this task under `GSSoC'23`

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Summary
- This PR is about optimizing image assets from `jpg`, `png` to `webp` conversion. 
- Optimizing images is crucial part of every site which plays an important role in site performance
- Along with optimization of images i modified the image source path in necessary `html` & `css` file to ensure proper working of this site after optimization
- You can compare the unoptimized & optimized image on the below screenshots, over `70%` of size reduction with optimization

## Type of change
_Please select the relevant option._
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation update
- [x] Optimization & some modification

## Checklist
_Please tick the relevant options._
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Screenshots

|  BEFORE: 2.54MB  | AFTER: 589KB  |
| :---: | :---: |
| Screenshot_2023-05-30-07-41-34_1366x768 | Screenshot_2023-05-30-07-42-32_1366x768 |

## Reviewers
- @Neilblaze

### CODE REVIEW

**Review by Neilblaze — APPROVED** [review]
LGTM!

**Comment by 0xabdulkhalid** [comment]
Hey there @Neilblaze, thank you for considering to merge this PR to optimize site images.

Here are the few things to consider:
- Adding a `GSSoC'23` & `level 2` label for this mid level task
- And assigning me to this PR so that i could keep track of my progress

File: img/vectors/script.2.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+jpg_files=$(find "$current_dir" -type f -name "*.jpg")
+
+# Loop through the jpg files and convert them to WebP format
+for jpg_file in $jpg_files; do
+
+  # Get the name of the WebP file
+  webp_file=${jpg_file%.jpg}.webp
+
+  # Convert the jpg file to WebP format
+  cwebp -q 40 "$jpg_file" -o "$webp_file"
```
[the remark this answers is not in the document]
Comment (0xabdulkhalid) [reply img/vectors/script.2.sh]: There's no major difference between `40` to `75`, the size can be `20kb` longer if it in `75`.

But regardless of it compression level `40` is enough to optimize

**Comment by Neilblaze** [comment]
@0xabdulkhalid This PR is tagged with issue #[ID](https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/issues/241)'s and involves no external collaborators, hence it's extraneous to reassign. Moreover, I need to confirm with @AbhiPatel10 if this is a `level 2` PR before adding the label, since this involves non-breaking changes & includes performance optimizations (refer to [this](https://cdn.discordapp.com/attachments/1099745008157986889/1112784850131357706/image.png) under Level1/Bullet3).

You can track your progress in this particular project by monitoring [this](https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/pulls?q=is%3Apr+author%3A0xabdulkhalid). Thanks!

File: img/vectors/script.2.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (Neilblaze) [reply img/vectors/script.2.sh]: That's the reason why I mentioned "nit".

**Comment by 0xabdulkhalid** [comment]
Wow @Neilblaze, thanks for such an insightful reply on my doubts.

PS: Can you share the document/screenshot of the `level` assignment, So that i can able to make issues to match the stated criteria.

**Comment by 0xabdulkhalid** [comment]
Bro @Neilblaze, the label `Level 1` is not valid.

Please change it to `level1` this only accepted by the Gssoc Backend. so that it can count this contribution otherwise this can't be counted

**Comment by Neilblaze** [comment]
Done ✔️

**Pull request merged**